mainloop.lua

```diff
@@ -689,16 +694,25 @@ function main_net_vs_lobby()
         end
       end
 
+      local function playerRatingString(playerName)
+        local rating = ""
+        if playerRatingMap and playerRatingMap[playerName] then
+          rating =  " (" .. playerRatingMap[playerName] .. ") "
```

Comment:
Remove extra space at the end; The " vs " string already takes care of that space
```suggestion
          rating =  " (" .. playerRatingMap[playerName] .. ")"
```